test/k8s-integration/cluster.go

```diff
@@ -307,6 +307,9 @@ func clusterUpGKE(gceZone, gceRegion string, numNodes int, imageType string, use
 				NodeConfig: &container.NodeConfig{
 					MachineType: "n1-standard-2",
 					ImageType:   imageType,
+					OauthScopes: []string{
+						"https://www.googleapis.com/auth/devstorage.read_only",
```

Comment:
I just remembered that https://www.googleapis.com/auth/cloud-platform is the uber-scope that everyone seems to be moving to these days.

There more discussion about this here: https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#using

> Generally, you can just set the cloud-platform access scope to allow access to most of the Cloud APIs, then grant the service account only relevant IAM roles. The combination of access scopes granted to the virtual machine instance and the IAM roles granted to the service account determines the amount of access the service account has for that instance. The service account can execute API methods only if they are allowed by both the access scope and its IAM roles.